jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml

```diff
@@ -81,6 +81,16 @@
                GM<X>+up (official GM plus Cloud-Updates)
                cloud9MX (cloud 9 milestones)
 
+      - bool:
+
+          name: prepare_install_crowbar
```

Comment:
I would recommend not adding another job parameter, but rather running this step always. Adding more complexity to these jobs is something we tried to avoid as much as possible in the past.

Reply:
Ok good point. I removed that option again in the most recent version of this PR.